events.cpp

```diff
@@ -28,13 +28,14 @@
 #include <map>
 #include <algorithm>
 #include "gamedata.h"
+#include <sstream>
```

Comment:
I haven't seen this std header yet, what does it give us and how important is it?

Reply:
This is stringstream, basically a buffer that allows easely to construct large strings.

I'm going to use that in world events too. Will refactor them a bit.